src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,42 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.dao.CarDaoImpl;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.dao.DriverDaoImpl;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerDao manufacturerDao = new ManufacturerDaoImpl();
+        Manufacturer bmwManufactured = new Manufacturer();
```

Comment:
```suggestion
        Manufacturer bmwManufacturer = new Manufacturer();
```
let's use a noun as a variable name